tests/specs/scoped.js

```diff
@@ -82,6 +82,58 @@ test('scoped Store .find()', function (t) {
   .catch(t.fail)
 })
 
+test('scoped Store .find(array) items not found', function (t) {
+  t.plan(2)
+
+  var store = new Store('test-db-scoped-find', merge({remote: 'test-db-scoped-find'}, options))
+  var testStore = store('test')
+
+  var testItems = [{ foo: 'bar1' }, { foo: 'bar2' }]
```

Comment:
I’d suggest that you set the `id` property in the `testItems` array (you can use `id` instead of `foo` if you like). Then you know the IDs already and don’t need to read them out from what `.find()` resolves with